hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -177,6 +177,12 @@
       .withDocumentation("Determines how to handle updates, deletes to file groups that are under clustering."
           + " Default strategy just rejects the update");
 
+  public static final ConfigProperty<String> SCHEDULE_INLINE_CLUSTERING = ConfigProperty
+      .key("hoodie.clustering.schedule.inline")
+      .defaultValue("false")
+      .withDocumentation("When set to true, clustering service will be attempted for inline scheduling after each write. Users have to ensure "
```

Comment:
I think we should add more details regarding how it interacts with (or differs from) other inline/async configs otherwise it might confuse users why we have separate configs. First, we could mention what will happen if this config is enabled together with the existing inline/async compaction/clustering configs. Secondly, we can use the corresponding config keys in the doc to be more explicit.